cross/rsnapshot/Makefile

```diff
@@ -0,0 +1,21 @@
+PKG_NAME = rsnapshot
+PKG_VERS = 1.4.2
+PKG_EXT = tar.gz
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_NAME = $(PKG_DIR).$(PKG_EXT)
+PKG_DIST_SITE = http://rsnapshot.org/downloads
+
+HOMEPAGE = http://rsnapshot.org/
+COMMENT  = Rsnapshot is a filesystem snapshot utility. It can take incremental snapshots of local and remote filesystems. 
+LICENSE  = GNU-GPL
+
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = --prefix=$(INSTALL_PREFIX) --with-perl=/usr/bin/perl --with-rsync=/usr/bin/rsync --with-ssh=/usr/bin/ssh
```

Comment:
`--prefix=$(INSTALL_PREFIX)` should already be set via configure.mk. See https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.configure.mk#L40. In other words, it should work without explicitly defining the prefix here.